app.py

```diff
@@ -36,7 +22,7 @@ def get_hub_by_id(id):
 
 @app.route('/hubs', methods=['POST'])
 def add_hub():
-    hub = Hub(id=request.json['id'], name=request.json['name'], user_number=request.json['user_number'])
```

Comment:
is the db field auto-incrementing for this to work?

Reply:
Yes